### PR TITLE
Update remote-help.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note to the document stating targeting a group of users will not be successful. 

fixes#https://github.com/MicrosoftDocs/memdocs/issues/3553

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/remote-actions/remote-help.md [hunk 1]
```diff
@@ -186,6 +186,9 @@ To opt out of automatic updates, specify enableAutoUpdates=0 as part of the inst
 
 6. Complete creation of the Windows app to have Intune deploy and install Remote Help on applicable devices.
 
+> [!NOTE]
+> Remote Help only works when targeting a group of devices, and that targeting a group of users will not be successful.
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/remote-help.md:190]:
```suggestion
> Remote Help only works when targeting a group of devices. Targeting a group of users will not be successful.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Smritib17 Please signoff this PR. Thanks

**Comment by yogkumgit** [comment]
@Smritib17 Gentle reminder. Thanks

**Comment by Smritib17** [comment]
This issue has been fixed with PR: https://github.com/MicrosoftDocs/memdocs-pr/pull/10208
Closing this PR as there were build errors.

**Pull request closed without merging**